style/style.css

```
@@ -13,6 +13,13 @@
     width: 100%;
 }
 
+@media(max-width: 1140px) {
```

А почему именно для рамера 1140px?
```suggestion
@media(max-width: 1140px) {
```

---

потому что ширина container 1110px, добавив внешние отступы по 15px справа и слева получаем ширину 1140px